includes/class-wc-payments-account.php

```diff
@@ -186,6 +186,17 @@ public function get_fees() {
 		return isset( $account['fees'] ) ? $account['fees'] : [];
 	}
 
+	/**
+	 * Gets the account live mode value.
+	 *
+	 * @return bool          Account is_live value.
+	 * @throws API_Exception Bubbles up from get_cached_account_data.
+	 */
+	public function get_is_live() {
+		$account = $this->get_cached_account_data();
+		return $account['is_live'];
```

Comment:
That's neat! I'd suggest making more strict though:
```php
public function get_is_live(): bool {
		$account = $this->get_cached_account_data();
		return $account['is_live'] ?? true;
}
```

Reply:
Sorry, I didn't see this and changed it as part of https://github.com/Automattic/woocommerce-payments/pull/972/commits/208444af57a35beaad5f0034f7de0186d80acb44 and added a few checks in https://github.com/Automattic/woocommerce-payments/pull/972/commits/e6b67c24ddef538f9d552daa0d8d1f0d49895c39. What do you think?